kitsune/forums/jinja2/forums/base.html

```diff
@@ -2,6 +2,9 @@
 {% if not scripts %}
   {% set scripts = ('forums',) %}
 {% endif %}
+{% if not ga_content_group %}
```

Comment:
I would remove the ga_content_group for all base pages. If one of them ends up in GA then we clearly have a bug. I don't think we should track any of those

Reply:
Will do!